### PR TITLE
Oliver

### PR DESCRIPTION
**Pull request by olivermeng6**
### Clean and refactor
- working on current implementation of `index.rmd`
- have not tested on `index.html` but should be the same
- have not done coded testing 
- potential changes listed as TODOs
- should be able to easily adapt to the version submitted by `quangv49`

### CODE REVIEW

**Comment by olivermeng6** [comment]
@dajmcdon

File: ploting.R
Comment (dajmcdon) [inline ploting.R:8]: for the sake of readability, better as 
```
if (scale == "ps") {
```

Comment (dajmcdon) [reply ploting.R:8]: Same in other places.

File: ploting.R
Comment (dajmcdon) [inline ploting.R:4]: should probably document these, or even better make the options more descriptive.
So, rather than `scale = c("s", "ls", "ps")`, you would use `scale_y = c("none", "log_scale", "per_capita")` and rather than `by = ...` you could say `subgroups = c("all", "health_authority", "age")`

File: ploting.R
Comment (dajmcdon) [inline ploting.R:53]: For some reason, most editors like an empty line at the end.

**Review by dajmcdon — CHANGES_REQUESTED** [review]

**Comment by dajmcdon** [comment]
Thanks Oliver! I'll merge this first after the changes, but then, let's incorporate this function into #7 before that one gets merged.

**Comment by olivermeng6** [comment]
@dajmcdon should I always tag you when I make changes?

**Review by dajmcdon — APPROVED** [review]
@olivermeng6 Looks great! Thank you. I haven't managed to make build checks run on a PR, so we'll find out if it works on merge.

**Review by dajmcdon — CHANGES_REQUESTED** [review]
However, there are a number of conflicts with `index.Rmd`. On your local branch, can you 
1. Checkout `main`
2. Pull
3. Checkout `oliver`
4. `git merge main`

At that point, it will ask you to resolve the conflicts. It should be pretty straightforward. Then commit and push. This PR will update automatically and I can merge.

Let me know if you have questions, or that's not clear.

**Comment by dajmcdon** [comment]
Oh, before you checkout `main`, you'll have to Sync your fork on Github. There should be a button at the top.

**Comment by olivermeng6** [comment]
@dajmcdon Hi Daniel, I believe the conflict is revolved. I didn't know how to do this previously. I also noticed a few warnings that arose, I will talk to Quang and fix those as well.

**Pull request closed without merging**
